### ISSUE TITLE
Results from python secir examples are not interpolated

### ISSUE THREAD

**Issue by HenrZu** [DLR-SC/memilio#442]
When running a simulation of a model in python, like `pycode/examples/simulation/secir_simple.py`,
the results are only available on the steps from the numerical integrator.

It would be really helpful if the results are interpolated over the given time interval.
Therefore, we should use the existing interpolation implementation from the c++ backend and include it via pybinds.

**Comment by mknaranja** [comment]
Please add labels

**Comment by mknaranja** [comment]
I had given this highest priority but let's coordinate with #447 to not (re)write it twice

**Comment by HenrZu** [comment]
The interpolation for the Secir model was already integrated in the bindins.
https://github.com/DLR-SC/memilio/blob/910b1af66848e28ce385906425635638423e5011/pycode/memilio-simulation/memilio/simulation/secir.cpp#L147
In PR #452, I have integrated them into the secir examples. For the Seir model I added the interpolation function and include it to the example as well. There should be no conflicts with #446.
